source/jormungandr/jormungandr/realtime_schedule/siri_lite.py

```diff
@@ -112,8 +112,15 @@ def _get_passages(self, route_point, resp):
             next_passages = []
             for next_expected_st in schedules:
                 # for the moment we handle only the NextStop and the direction
-                dt = self._get_dt(next_expected_st['monitoredVehicleJourney']['monitoredCall']['expectedDepartureTime'])
-                direction = next_expected_st.get('destinationName', {}).get('value')
+                expected_dt = next_expected_st.get('monitoredVehicleJourney', {}).get('monitoredCall', {}).get('expectedDepartureTime')
+                if not expected_dt:
+                    continue
+                dt = self._get_dt(expected_dt)
+                destination = next_expected_st.get('monitoredVehicleJourney', {}).get('destinationName', {})
```

Comment:
if it's a list, can you put the default value to `[]` ? it's a bit confusing as it is